src/test/java/com/github/princesslana/eriscasper/repository/event/TestChannelsFromEvents.java

```diff
@@ -34,7 +34,11 @@ public void getChannel_whenGuildCreated_shouldCacheChannel() {
     TestObserver<Channel> observer = new TestObserver<>();
 
     Guild guild = simpleCreateGuild();
-    Channel channel = guild.getChannels().get(0);
+    Channel channel =
+        guild
+            .getChannels()
+            .orElseThrow(() -> new IllegalStateException("Illegal state of list."))
```

Comment:
Since this is testing, doing a `get().get(0)` is ok. It'll throw an exception anyway.

Reply:
Alright, fair enough, just hate the yellow warnings that pop up for it. I'll change it.